.github/workflows/cron-check-usage.yml

```
@@ -0,0 +1,37 @@
+          http_code=$(echo "$response" | tail -n1)
+          body=$(echo "$response" | sed '$d')
+          
+          if [ "$http_code" -eq 200 ]; then
+            echo "✅ Usage check completed successfully"
+            echo "$body" | jq '.' || echo "$body"
+          else
+            echo "❌ Usage check failed with HTTP $http_code"
+            echo "$body"
+            exit 1
+          fi
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)